packages/graphql/src/translate/queryAST/ast/operations/ReadOperation.ts

```diff
@@ -227,20 +240,49 @@ export class ReadOperation extends Operation {
         };
     }
 
+    private hasCypherSort() {
+        return (
+            this.sortFields.filter((s) => {
+                return s instanceof CypherPropertySort;
+            }).length > 0
+        );
+    }
+
     public getChildren(): QueryASTNode[] {
         return filterTruthy([...this.filters, this.authFilters, ...this.fields, this.pagination, ...this.sortFields]);
     }
 
     protected getFieldsSubqueries(context: QueryASTContext): Cypher.Clause[] {
+        const nonCypherFields = this.fields.filter((f) => {
+            const isCypher = f instanceof CypherAttributeField;
+            return !isCypher;
+        });
+
+        return filterTruthy(
+            nonCypherFields.flatMap((f) => {
+                return f.getSubqueries(context);
+            })
+        ).map((sq) => {
+            return new Cypher.Call(sq).innerWith(context.target);
+        });
+    }
```

Comment:
As we filters anyway the returned array, we can simplify
```suggestion
    protected getFieldsSubqueries(context: QueryASTContext): Cypher.Clause[] {
        return filterTruthy(
            this.fields.flatMap((f) => {
                if (f instanceof CypherAttributeField) {
                    return;
                }
                return f.getSubqueries(context);
            })
        ).map((sq) => {
            return new Cypher.Call(sq).innerWith(context.target);
        });
    }

```